packages/tradershub/src/features/cfd/flows/OtherCFDs/Dxtrade/AvailableDxtradeAccountsList/AvailableDxtradeAccountsList.tsx

```diff
@@ -25,14 +27,23 @@ const LeadingIcon = () => (
     </div>
 );
 
-const TrailingButton = () => <TradingAccountCardLightButton />;
+const AvailableDxtradeAccountsList = () => {
+    const { show } = Provider.useModal();
+    const { setCfdState } = Provider.useCFDContext();
 
-const AvailableDxtradeAccountsList = () => (
-    <TradingAccountCard leading={LeadingIcon} trailing={TrailingButton}>
-        <TradingAccountCardContent title={PlatformDetails.dxtrade.title}>
-            This account offers CFDs on a highly customisable CFD trading platform.
-        </TradingAccountCardContent>
-    </TradingAccountCard>
-);
+    const TrailingButton = () => <TradingAccountCardLightButton onSubmit={trailingButtonClick} />;
+
+    const trailingButtonClick = () => {
+        setCfdState('platform', PlatformDetails.dxtrade.platform);
```

Comment:
```suggestion
        setCfdState('platform', CFDPlatforms.DXTRADE);
```

Reply:
for consistency purpose, I think it is better to use `PlatformDetails` throughout since it has more complete information that is used on other parts of the PasswordModal, else, there will be `CFDPlatforms` in one file and `PlatformDetails` on another which is messy

Reply:
Agreed with @shontzu-deriv 
The const should be consistent, we should use only one.
cc: @mahdiyeh-deriv 
<img width="601" alt="image" src="https://github.com/binary-com/deriv-app/assets/120543468/330c7f88-afd1-48d1-b097-25386a230445">